compiler/src/dmd/statement.d

```diff
@@ -82,15 +82,6 @@ extern (C++) abstract class Statement : ASTNode
         return b;
     }
 
-    override final const(char)* toChars() const
```

Comment:
This change is kind of problematic because here toChars is overriding ASTNode.toChars() which simply asserts. This means that uses of s.toChars() will assert even if hdrgen.d is imported.